index.php

```diff
@@ -1,27 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Template</title>
+    <title>Matt's EPL Card Collection</title>
 
     <meta name="description" content="Template HTML file">
     <meta name="author" content="iO Academy">
 
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
     <link rel="stylesheet" href="css/styles.css">
 
-    <link rel="icon" href="images/favicon.png" sizes="192x192">
-    <link rel="shortcut icon" href="images/favicon.png">
-    <link rel="apple-touch-icon" href="images/favicon.png">
+    <link rel="icon" href="images/eplLogo.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/eplLogo.png">
+    <link rel="apple-touch-icon" href="images/eplLogo.png">
 
     <script defer src="js/index.js"></script>
+    <?php
+    require_once 'src/Navbar.php';
+    require_once 'src/Entities/PlayerCard.php';
+    require_once 'src/Models/PlayerCardModel.php';
+
+
+
+    $db = new PDO('mysql:host=db; dbname=Collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $playerCards = new PlayerCardModel($db);
+    $allCards = $playerCards->getAllCards();
+    ?>
 </head>
 
 <body>
+    <ul class='playersTable'>
+        <strong>
+            <li>Player ID</li>
+        </strong>
+        <strong>
+            <li>Player Name</li>
+        </strong>
+        <strong>
+            <li>Players Club</li>
+        </strong>
+        <strong>
+            <li>Players Position</li>
+        </strong>
+        <strong>
+            <li>Players Defence Score</li>
+        </strong>
+        <strong>
+            <li>Players Control Score</li>
+        </strong>
+        <strong>
+            <li>Players Attack Score</li>
+        </strong>
+        <strong>
+            <li>Players Total Score</li>
+        </strong>
 
-<h1>Website Template</h1>
+        <?php
+        foreach ($allCards as $card) {
```

Comment:
You should make a function that returns all the code needed to display the cards, you might end up re-using it on the deleted page in future. 

Also good practice to make into a function even if you don't end up using it again, just incase. 

This function would also require some unit tests, so some good practice there!